### PR TITLE
This branch is for creating contact form in the mobile version.

### PR DESCRIPTION
**Pull request by thenarain**
Add files:

1. Update Markup
2. Update styles
3. Add Images
4. Use Formspree to submit the form

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -290,5 +290,60 @@ <h3>Skills</h3>
...
+                name="user_message"
+                placeholder="Write your message here"
+                maxlength="500"
+                required
+              ></textarea>
+            </li>
+            <li class="button-list">
+              <button class="button1" type="submit">Get It Touch</button>
+            </li>
+          </ul>
```
Comment (aimemalaika) [inline index.html:344]: - [ ] Kindly use flex properties to layout the elements.
- [ ] Kindly add some button interactions such as hover, active (when it's pressed), disabled, and enable (default state) are styled as shown in the Figma template.

**Review by aimemalaika — CHANGES_REQUESTED** [review]
Hi @thenarain, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

the-simpsons-homer-simpson

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (thenarain) [reply index.html:344]: review1
I have used flexbox for the layout.
hover
I have used hover also. It is just that during inspect it is not visible.

**Review by Ismailco — APPROVED** [review]
Hi @thenarain,


cr-approve

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**